### ISSUE TITLE
No result of SUPA On-demand Analysis when querying function pointers

### ISSUE THREAD

**Issue by legendtang** [SVF-tools/SVF#341]
Hi,

For the following bitcode file, I ran SUPA on-demand points-to analysis and found there is no callsite result.

[imagick_r.0.4.opt.bc.zip](https://github.com/SVF-tools/SVF/files/5430322/imagick_r.0.4.opt.bc.zip)

Step to reproduce:
Run `dvf --dfs --query=funptr --stat imagick_r.0.4.opt.bc`, it shows the following result

```
=================================================
Total virtual callsites: 0
Total analyzed virtual callsites: 0
Indirect call map size: 80
Precise callsites: 0
Zero target callsites: 0
One target callsites: 0
Two target callsites: 0
More than two target callsites: 0
=================================================
```

If printing the Anderson analysis results of call sites using `print-fp`, it does have many. According to the [paper](https://yuleisui.github.io/publications/tse18.pdf), there should be a fallback to Anderson right?

**Comment by yuleisui** [comment]
What are Andersen's results? It looks strange to me.

**Comment by yuleisui** [comment]
Did you print all the pts of `dvf -dfs --query=funptr `?

**Comment by legendtang** [comment]
Yes, I print all function pts of that command by `dvf --dfs --query=funptr --stat`. Here is the log
[dvf.log](https://github.com/SVF-tools/SVF/files/5433209/dvf.log)

**Comment by yuleisui** [comment]
It looks to me that dvf analyzes all the indirect calls.  You can print all the pts. It should be there.
